### DISCUSSION TITLE
Better Calendar View

### DISCUSSION THREAD

**Discussion by 2011413125** [iSoron/uhabits#1287] [Feature Requests]
The current calendar view does not seem natural as paper or digital calendars. 

A better calendar view gives a better experience and outlook at the habit on monthly and yearly basis. I think this kind of visual creates better impact than numerical stats.

Here is Loop Calendar

![loophabit](https://user-images.githubusercontent.com/1507526/151794692-5e4e849d-d215-4604-a734-07793744adf8.jpg)

Here is ezHabit ( or consider any other you habit app like habitnow etc.)

![ezhabit-month](https://user-images.githubusercontent.com/1507526/151794728-b6dfd3ac-544b-492e-9eda-1f960687ede7.jpg)

![ezhabit-year](https://user-images.githubusercontent.com/1507526/151794738-01893740-1745-4cdc-945a-da9b9e87b7db.jpg)

Thanks.

**Comment by iSoron** [comment]
Thank you for the suggestion, @2011413125. Our current calendar view was inspired by the one on GitHub, and it's just meant to provide a compact view of your habit history. Showing a proper calendar would probably take too much space on the habit overview screen.

**Reply by 2011413125** [reply]
I do understand that. But its no where natural. The satisfaction just looking at all the greens(done mark) on the calendar seems far greater.
 
Here is a mockup. Show 3 month scrollable calendar. And on either edit or view year. The current Year.

![loop-year-month-mockup](https://user-images.githubusercontent.com/1507526/152214318-412473b2-ed80-4d71-9b8f-6d26629d369b.png)
